ReactAndroid/build.gradle

```diff
@@ -28,7 +28,7 @@ task createNativeDepsDirectories {
 
 task downloadBoost(dependsOn: createNativeDepsDirectories, type: Download) {
     // Use ZIP version as it's faster this way to selectively extract some parts of the archive
-    src 'https://unpkg.com/boost-mirror@1.57.0/boost.zip'
+    src 'https://downloads.sourceforge.net/project/boost/boost/1.57.0/boost_1_57_0.zip' 
```

Comment:
Why revert it? @mkonicek landed this commit (https://github.com/facebook/react-native/commit/1488725db38e925e096df94e5ca8d9ac712854db) days ago...